src/scripts/db.js

```diff
@@ -42,9 +42,9 @@ var Db = {
     }
 
     if (!TogglOrigins[origin]) {
-      // Handle cases where subdomain is used (like web.any.do, we remove web from the beginning)
+      // Handle cases where subdomain is used (like web.any.do (or sub1.sub2.any.do), we remove web from the beginning)
       origin = origin.split(".");
-      origin.shift();
+      while (origin.length > 2) { origin.shift(); }
```

Comment:
Still not letting you off the hook. I believe this will fail with SLD-enabled domains such as `.co.uk`.
https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains shows there's a hefty number of such TLDs.
The only catch-all solution I can think of is to check `TogglOrigins` at each iteration and hope that we never introduce an origin that is just `com` or `co.uk`.